### ISSUE TITLE
bugs what i noticed

### ISSUE THREAD

**Issue by joksik** [lirios/lirios#37]
Hello i use latest iso from 03.06.2017 i have old laptop so i dont use uefi boot only legacy boot

1) after livecd start i dont have any mouse cursor on desktop only blue wallpaper and icons. after moving mouse/touchpad like crazy finally cursor appeared so i could test another things

2) in taskbar (right down corner) there is no icon for network only empty space between icons. when i click this empty space i saw network options and i can connect without problem to my home wifi so network works good only icon on taskbar is missing

3) accident i click on left down hotcorner (i was try click on menu)  and all system freeze - my desktop minimized and centered so i think it`s hotcorner function like "show all window"s but i can`t exit from this only restart my computer works

**Comment by majorgreys** [comment]
I tried Liri for the first time this week on my desktop (installing packages on an existing Arch system) and laptop (booting off a usb stick). In both cases I had the same problem as @joksik had with the mouse cursor being missing. I suppose this is a recently introduced bug as I can't seem to find any issues online with the mouse cursor on Liri.

**Comment by joksik** [comment]
i read Liri road-map. they say that release 1.0 (stable ?) will be in the end of 2017 ;) it`s very promising project so i will test next release and report bug that i found. i have hope that they will fix it in the future

**Comment by joksik** [comment]
any news about new nightly version? maybe there will be fixed some reported bugs

**Comment by plfiorini** [comment]
@joksik

1. is done, mouse cursor is now centered on the primary screen at startup
3. fixed, hitting the hot corner again wil exit the effect

**Comment by joksik** [comment]
ok thx for fix this reported issue. i will test it with next release

ps: changing wallpapers is not ready ?

**Comment by joksik** [comment]
@plfiorini 

i tested latest night image from 19.06.2017 and change background bug and freeze hot-corner bug was fixed but still:

1) i dont have mouse cursor. i must open settings and then mouse cursor is start working
2) no icon for network settings only empty black space between icons in right down corner

anyway thanks for your hard work!

**Comment by plfiorini** [comment]
yeah i have noticed that as well with vmware, can't reproduce on bare metal.

**Comment by joksik** [comment]
ok i have this issue on 2 computers so maybe i can help? what logs i must attach? or what i must send you to help with this? i remember that image from april and may was works
